Kitodo/src/test/java/org/kitodo/MockDatabase.java

```diff
@@ -824,10 +824,10 @@ private static void insertProjectFileGroups() throws DAOException, DataException
         fifthProjectFileGroup.setProject(project);
         serviceManager.getProjectFileGroupService().saveToDatabase(fifthProjectFileGroup);
 
-        project.getProjectFileGroups().add(fifthProjectFileGroup);
-        project.getProjectFileGroups().add(fifthProjectFileGroup);
-        project.getProjectFileGroups().add(fifthProjectFileGroup);
-        project.getProjectFileGroups().add(fifthProjectFileGroup);
+        project.getProjectFileGroups().add(firstProjectFileGroup);
```

Comment:
This changes did not fit in this commit nor pull request. Can you please move it to a separate commit or better pull request?